packages/apps/src/overlays/Attest.tsx

```diff
@@ -0,0 +1,49 @@
+// Copyright 2017-2020 @polkadot/apps authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import React from 'react';
+import { useApi, useAccounts, useCall } from '@polkadot/react-hooks';
+import { Option } from '@polkadot/types';
+import { EthereumAddress } from '@polkadot/types/interfaces';
+
+import { Link } from 'react-router-dom';
+
+import { useTranslation } from '../translate';
+import BaseOverlay from './Base';
+
+interface Props {
+  className?: string;
+}
+
+function Attest ({ className }: Props): React.ReactElement<Props> | null {
+  const { t } = useTranslation();
+  const { allAccounts } = useAccounts();
+  const { api } = useApi();
+
+  // Find accounts that need attest. They are accounts that
+  // - already preclaimed,
+  // - didn't sign the attest yet.
+  // `claims.preclaims` returns Some() for these accounts.
+  const needAttest = useCall<[Option<EthereumAddress>]>(api.query?.claims?.preclaims?.multi, [allAccounts])?.filter((opt) => opt.isSome);
```

Comment:
```suggestion
  const needAttest = useCall<[Option<EthereumAddress>]>(isApiReady && api.query.claims?.preclaims?.multi, [allAccounts])?.filter((opt) => opt.isSome);
```

Consistency with elsewhere. Since this (or a preferred counter alignment), is outside of content it needs the ready check - `useCall` does handle `false` correctly in this case, doing nothing.